front/ui/README.md

```diff
@@ -10,8 +10,7 @@ Development will leverage preconstruct. **Documentation yet to come**.
 
 ## Publishing versions
 
-In a nutshell: we consider our version numbers metadata of the project. We use
-[Semantic Versioning](https://semver.org/).
+osrd-ui releases are published automatically together with OSRD releases. osrd-ui's latest version is always in sync with OSRD.
```

Comment:
This might affect our external users :thinking: But I remember the discussions about it :+1:

Reply:
Yeah. We can change how releases are managed later on if needed.